File: src/main/java/dev/lonami/uniffidl/psi/UdlElementType.java
package dev.lonami.uniffidl.psi;

import com.intellij.psi.tree.IElementType;
import dev.lonami.uniffidl.UdlLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class UdlElementType extends IElementType {
    public  UdlElementType(@NonNls @NotNull String debugName) {
        super(debugName, UdlLanguage.INSTANCE);
    }
}

File: src/main/kotlin/dev/lonami/uniffidl/psi/UdlElementType.kt
package dev.lonami.uniffidl.psi

import com.intellij.psi.tree.IElementType
import dev.lonami.uniffidl.UdlLanguage
import org.jetbrains.annotations.NonNls

class UdlElementType(@NonNls debugName: String) : IElementType(debugName, UdlLanguage.INSTANCE)
